app/components/main/main.component.ts

```diff
@@ -158,7 +168,7 @@ export class MainComponent implements OnInit, MenuEventListener, REPLDelegate, T
     menuEvent(e: MenuEvent) {
         switch (e) {
             case MenuEvent.NEW_FILE:
-                this.newFile();
+                this.promptNewFile();
```

Comment:
Something like [this](https://github.com/2graphic/sinap-design-docs/blob/master/ui_sketches/blank_canvas.png)?